iOS Client/HTTPClient/MBClient.swift

```diff
@@ -40,6 +43,20 @@ class MBClient: NSObject {
         case failedPagingRequest(msg: String)
     }
     
+    public enum PodcastStream: String {
+        case pz = "http://pzspodcast.fireside.fm/rss"
```

Comment:
can we reuse the fields defined above?

Reply:
I notice there's an inconsistency with http vs https for mockingCast. It should be https, same goes for pz

Reply:
https works for the fireside endpoints, we should add https to our url constants for fireside ones